.github/labeler.yml

```diff
@@ -118,15 +118,17 @@ units:
 
 utils:
   - cextern/expat/**/*
-  - any: ['**/utils/**/*', '!astropy/utils/iers/**/*', '!docs/utils/iers.rst']
+  - any: ['**/utils/**/*',
```

Comment:
@pllim - do these additional changes to the labeller make sense? Thanks!

Reply:
I think that's correct.

p.s. 100 commits and `+5,427 −112` 😱

Reply:
Yes, this was quite a bit of work. Though most of the lines is somewhat boilerplate in `function_helpers` and lots of tests...